code/modules/surgery/tools.dm

```diff
@@ -330,3 +330,74 @@
 /obj/item/surgicaldrill/advanced/examine()
 	. = ..()
 	. += " It's set to [tool_behaviour == TOOL_DRILL ? "drilling" : "mending"] mode."
+
+/obj/item/shears
+	name = "amputation shears"
+	desc = "A type of heavy duty surgical shears used for achieving a clean separation between limb and patient. Keeping the patient still is imperative to be able to secure and align the shears."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "shears"
+	flags_1 = CONDUCT_1
+	item_flags = SURGICAL_TOOL
+	toolspeed  = 1
+	force = 12
+	w_class = WEIGHT_CLASS_NORMAL
+	throwforce = 6
+	throw_speed = 2
+	throw_range = 5
+	custom_materials = list(/datum/material/iron=8000, /datum/material/titanium=6000)
+	attack_verb = list("sheared", "snipped")
+	sharpness = IS_SHARP
+	custom_premium_price = 1800
+
+/obj/item/shears/attack(mob/living/M, mob/user)
+	if(!iscarbon(M) || user.a_intent != INTENT_HELP)
+		return ..()
+
+	if(user.zone_selected != BODY_ZONE_PRECISE_GROIN && user.zone_selected == BODY_ZONE_CHEST)
+		return ..()
+
+	var/mob/living/carbon/patient = M
+
+	if(HAS_TRAIT(patient, TRAIT_NODISMEMBER))
+		to_chat(user, "<span class='warning'>The patient's limbs look too sturdy to amputate.</span>")
+		return
+
+	var/candidate_name
+	var/obj/item/organ/tail_snip_candidate
+	var/obj/item/bodypart/limb_snip_candidate
+
+	if(user.zone_selected == BODY_ZONE_PRECISE_GROIN)
+		tail_snip_candidate = patient.getorganslot(ORGAN_SLOT_TAIL)
+		if(!tail_snip_candidate)
+			to_chat(user, "<span class='warning'>[patient] does not have a tail.</span>")
+			return
+		candidate_name = tail_snip_candidate.name
+
+	else
+		limb_snip_candidate = patient.get_bodypart(check_zone(user.zone_selected))
+		if(!limb_snip_candidate)
+			to_chat(user, "<span class='warning'>[patient] is already missing that limb, what more do you want?</span>")
+			return
+		candidate_name = limb_snip_candidate.name
+
+	var/amputation_speed_mod
+
+	patient.visible_message("<span class='danger'>[user] begins to secure [src] around [patient]'s [candidate_name].</span>", "<span class='userdanger'>[user] begins to secure [src] around your [candidate_name]!</span>")
+	playsound(get_turf(patient), 'sound/items/ratchet.ogg', 20, TRUE)
+	if(patient.stat == DEAD || patient.stat != CONSCIOUS || patient.IsStun()) //Stun is used by paralytics like curare it should not be confused with the more common paralyze.

```

Comment:
if(patient.stat) means patient is not conscious, or to be specific in soft crit, unconscious or dead.

Reply:
I don't want the fast dismember to work on people who are soft critted. I will try to come up with a statement that makes more sense.

Reply:
Right, this already does though since CONSCIOUS is 0 and you are checking stat for != conscious. Why not soft crit?

Just if(patient.stat > SOFT_CRIT) would to do the trick.

```
#define CONSCIOUS	0
#define SOFT_CRIT	1
#define UNCONSCIOUS	2
#define DEAD		3
```

Reply:
I used patient.stat == DEAD || patient.stat == UNCONSCIOUS for since it though it was easier to understand at a glance.